docs/api-docs/tax/tax-rates-zones.md

```diff
@@ -0,0 +1,464 @@
+# Tax Rates and Tax Zones
+
+[Tax rates](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-rate) are percentages for which you tax products or services. Tax zones are geographic locations that you define in your store. [Tax zones](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-zone) allow you to vary tax rates for different customer groups and locations. You can account for multiple tax jurisdictions by adding multiple tax rates to a zone. Tax rates and zones apply for all storefronts on a store.
```

Comment:
> You can account for multiple tax jurisdictions by adding multiple tax rates to a zone

I'm not sure about this, I'd probably drop the sentence.

Multiple jurisdictions = multiple tax zones. Multiple tax rates only speaks to use cases such as compounding tax rates and offering a further, detailed breakdown.